bodhi/server/models.py

```diff
@@ -4106,7 +4100,7 @@ def update_summary(self):
             list: A list of dictionaries with keys 'alias' and 'title', indexing each update alias
                 and title associated with this Compose.
         """
-        return [{'alias': u.alias, 'title': u.beautify_title(nvr=True)} for u in self.updates]
+        return [{'alias': u.alias, 'title': u.get_title(nvr=True)} for u in self.updates]
```

Comment:
If `beautify=False` is the default, this should be `u.get_title(nvr=True, beautify=True)` instead.